src/modules/core/components/button/button.interface.ts

```diff
@@ -0,0 +1,17 @@
+import type { ButtonHTMLAttributes } from 'react';
+import type { IconName } from '@/modules/core/components';
+import type { StylingProps } from '@/styles/styles.types';
+
+export type ButtonVariant = 'primary' | 'secondary' | 'custom';
```

Comment:
How about using `textOnly` instead of `custom` variant, in that case, we will show that there is no styles for button, because custom in my opinion doesn't show how this button will be styled.

Reply:
`custom` means that button has only `.base` styles and expect that you put your own styles by using `StylingProps`. So, I think `textOnly` isn't 100% suitable because such button can also be "icon only". I can suggest `unstyled` or `custom`. What do you think?

Reply:
Yeah, let's make it `unstyled`. I doesn't sound super well in my opinion, but it show us what we need.